File: Proyectos Android/RecyclerViewPokemon/app/src/main/java/com/example/recyclerviewpokemon/MainActivity.java
package com.example.recyclerviewpokemon;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.recyclerviewpokemon.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(ActivityMainBinding.inflate(getLayoutInflater()).getRoot());
    }
}

File: Proyectos Android/app/src/main/java/com/example/retrofittutokotlin/MainActivity.kt
package com.example.retrofittutokotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.retrofittutokotlin.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(ActivityMainBinding.inflate(layoutInflater).root)
    }
}